t/e2e/whisper/whisper_jail_test.go

```diff
@@ -323,12 +325,12 @@ func (s *WhisperJailTestSuite) TestJailWhisper() {
 		for {
 			filter, err := cell.Get("filter")
 			r.NoError(err, "cannot get filter")
-			filterID, err := cell.GetObjectValue(filter, "filterId")
+			filterID, err := cell.GetObjectValue(filter.Value(), "filterId")
 			r.NoError(err, "cannot get filterId")
 
 			select {
 			case <-done:
-				ok, err := s.WhisperAPI.DeleteMessageFilter(filterID.String())
+				ok, err := s.WhisperAPI.DeleteMessageFilter(filterID.Value().String())
```

Comment:
```Go
--- FAIL: TestWhisperJailTestSuite (306.80s)
    --- FAIL: TestWhisperJailTestSuite/TestJailWhisper (306.80s)
	Error Trace:	whisper_jail_test.go:338
	Error:      	polling for messages timed out
```

Still sorting through this test failure...

Reply:
There is a few changes to this test in the upcoming PR, maybe one of these can help you?
https://github.com/status-im/status-go/pull/703/files#diff-d6812742358ce4e1b26dae5465453933

Reply:
GOMAXPROCs did the trick - thank you!